plasma_framework/package.json

```diff
@@ -8,11 +8,14 @@
   "dependencies": {
     "@truffle/hdwallet-provider": "^1.0.37",
     "dotenv": "^8.0.0",
+    "elliptic": ">=6.5.3",
     "lodash.clonedeep": "^4.5.0",
     "openzeppelin-solidity": "2.3.0",
+    "solc": "^0.5.11",
```

Comment:
Do we need this change? I thought you have another PR for elliptic and I don't follow where this solc comes from?

Reply:
nop! my leftovers